src/main/java/taxi/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package taxi.service;
+
+import java.util.Optional;
+import taxi.exception.AuthenticationExeption;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverService driverService;
+
+    @Override
+    public Driver login(String login, String password) throws AuthenticationExeption {
+        Optional<Driver> driver = driverService.findByLogin(login);
+        if (driver.get().getPassword().equals(password)) {
```

Comment:
1. If we have empty optional on this stage we will get exception. Please add check to prevent it
2. if/else may be simplified:
if (someWrongCondition) {
    throw new Exception();
}
return result;